spec/compiler/data/build

```diff
@@ -0,0 +1 @@
+puts "this file is never compiled (#1412)"
```

Comment:
Meant to add a spec or just relying on the side effects breaking due to this from the other specs?